Core/Src/leds/rgb_led_handler.c

```diff
@@ -21,13 +9,21 @@
 #include "util/global_variables.h"
 #include "util/util.h"
 
-void write_rgb_color(rgb_t rgb_gpio);
-void write_debug_color(rgb_t rgb_gpio);
-rgb_t get_rgb_color(cores_t color);
-void blink_rgb(uint32_t delay);
+static void write_rgb_color(rgb_t rgb_gpio);
+// static void write_debug_color(rgb_t rgb_gpio);
+static void write_pattern(rgb_led_message_t message, int delay);
+static uint32_t are_messages_available(osMessageQueueId_t q_id);
+static void blink_rgb(uint32_t delay);
+rgb_t get_rgb_color(colors_t color);
 
-osStatus_t set_rgb_led(cores_t color, control_rgb_led_e control) {
-    rgb_led_message_t message = {color, control};
+osStatus_t set_rgb_led(const colors_t* pattern, control_rgb_led_e control,
+                       uint8_t sizeOfPattern) {
+    rgb_led_message_t message;
+    message.control       = control;
+    message.sizeOfPattern = sizeOfPattern;
```

Comment:
```suggestion
    message.size_of_pattern = size_of_pattern ;
```
nosso padrão é snake_case, não camelCase